File: testprojects/java/benchmark/accuracy-test-2013-04-17/domain/indirect/violatingfrom/CallInstanceMethodIndirect_MethodMethodViaConstructor.java
package domain.indirect.violatingfrom;

import domain.indirect.intermediate.BackgroundService;

public class CallInstanceMethodIndirect_MethodMethodViaConstructor {
	
	private String test;
	
	public CallInstanceMethodIndirect_MethodMethodViaConstructor() {
	}

	public String case1(){
		test = new BackgroundService().getServiceOne().getName();
		return test;
	}
}

File: src/test/resources/csharp/benchmark/Domain/Indirect/ViolatingFrom/CallInstanceMethodIndirect_MethodMethodViaConstructor.cs
namespace Domain.Indirect.ViolatingFrom
{
	public class CallInstanceMethodIndirect_MethodMethod_ViaConstructor
	{
		private string test;

		public void Case1()
		{
            test = new Domain.Indirect.Intermediate.BackgroundService().GetServiceOne().GetName();
		}
	}
}
